server/routes/plan-history/locale.json

```diff
@@ -19,7 +19,10 @@
       "didNotAgree": "{{ subject.givenName }} did not agree to this plan.",
       "couldNotAgree": "{{ subject.givenName }} could not agree to this plan."
     },
-    "ariaText": "View {{ hiddenText }}",
+    "ariaText": {
+      "ifLatestGoal": "View latest version of {{ hiddenText }}",
```

Comment:
Given that these variable names and the ones in linktype above serve the same purpose can we name them the same? e.g. ariaText.viewLatest & ariaText.viewGoal and linkType.viewLatest & linktype.viewGoal ?